lib/bolt/inventory/target.rb

```diff
@@ -176,13 +167,19 @@ def password
       end
 
       def options
-        transport_config.dup
+        transport_config
       end
```

Comment:
I'm not sure if this wants to return the `Bolt::Config::Transport` object or just its config hash. I'm leaning towards the former..

Reply:
That seems right to me. If we ever wanted higher-level transport-specific logic, it could live on the config class.